### PR TITLE
LAB | HTML & CSS: NPM Clone

### PR DESCRIPTION
**Pull request by Hoceritis**
First step, website architecture

### CODE REVIEW

**Comment by Hoceritis** [comment]
My version is far from perfect.
What I realised :
_ I have trouble still with flex box. The elements positionning on the page and between each other is something I need to dig.
_Html is much easier to understand and play around with than Css.

File: index.html [hunk 1]
```diff
@@ -1,9 +1,82 @@
...
+          <img id="npm" src="images/npm-logo.png" alt="logo" />
+          <form >
+              <input type="text" name="search" placeholder='Search packages'id="search-input">
+              <button type="submit">Search</button>
+          </form>
+          <div class="top-button">
+              <a href="#" id="join"> Join </a>
+              <a href="#" id="login"> Log In </a>
+          </div>
+      </div>
```
Comment (wassbeer) [inline index.html:34]: From this line on, some tags get mixed up and the indentation is off. 
Before this line, all looks fine.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+
+header > div {
+    padding: 0 25px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    border-bottom: 1px solid lightgray;
+  }
+  
+  nav {
```
Comment (wassbeer) [inline style.css:15]: Please be mindful about the indentation here as well.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,198 @@
+@import url('https://fonts.googleapis.com/css?family=Poppins');
+
+body {
+  font-family: 'Poppins';
+}
+
+header > div {
```
Comment (wassbeer) [inline style.css:7]:
```
header > div {
    padding: 10px 25px;
    display: flex;
    justify-content: space-between;
    align-items: center;
    border-bottom: 1px solid lightgray;
}
```

File: style.css [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+    width: 30px;
+    margin-right: 1rem;
+  }
+
+  #npm {
+    width: 55px;
+    margin-right: 1rem;
+  }
+
+  .search-bar form {
```
Comment (wassbeer) [inline style.css:42]:
```
.search-bar form {
    display: flex;
    flex-direction: row;
    flex-grow: 1;
    align-items: center;
    margin-block-end: 0em;
}
```

File: style.css [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+    display: flex;
+    flex-direction: row;
+    flex-grow: 1;
+}
+
+#search-input{
```
Comment (wassbeer) [inline style.css:48]:
```
#search-input {
    flex-grow: 1;
    background-color: rgba(0,0,0,.05);
    background-image: url(images/magnifying-glass.png);
    background-repeat: no-repeat;
    background-size: 26px;
    background-position: 15px center;
    padding-left: 50px;
    border: none;
    font-size: 1.2rem;
    height: 50px;
    margin-left: 5px;
}
```

**Review by wassbeer — COMMENTED** [review]
Hey Sullivan,

Good stuff!

I placed a few comments regarding the code indentation/formatting. Please take a look at it.

There are also still some differences between the clone and the example. I placed some CSS suggestions how to alter the styling in the header. You could think of making the cards smaller in section 3, in order to let them fit on the screen, centrally.

Please reach out if you have a question.

Thomas (TA WDFT)

**Pull request closed without merging**